rskj-core/src/main/java/co/rsk/net/NodeMessageHandler.java

```diff
@@ -247,20 +239,12 @@ private synchronized void sendStatusToAll() {
         this.channelManager.broadcastStatus(status);
     }
 
-    @CheckForNull
     public synchronized Block getBestBlock() {
-        if (this.blockProcessor != null)
-            return this.blockProcessor.getBlockchain().getBestBlock();
-
-        return null;
+        return this.blockProcessor.getBlockchain().getBestBlock();
```

Comment:
Eventually, we'll want a direct dependency on Blockchain :)

Reply:
Yes, we still have some dependencies to improve. This is a step.